### PR TITLE
Fruitties workflow

### PR DESCRIPTION
**Pull request by clivejefferies**
Added files to enable building of Fruitties.

This is a lot simpler than DiceRoller as the Podfile is only contains what is needed to build and is located within the iosApp folder. 

Also added building on feature branches

### CODE REVIEW

File: .github/workflows/Fruitties.yaml
Comment (bentrengrove) [inline .github/workflows/Fruitties.yaml:1]: Just need to update the name

**Review by bentrengrove — COMMENTED** [review]
Thanks for this! Just one small thing to fix up and we should be good

File: .github/workflows/Fruitties.yaml
Comment (clivejefferies) [reply .github/workflows/Fruitties.yaml]: Done

**Review by bentrengrove — APPROVED** [review]
LGTM! I guess we won't know if it works until I merge it, so I'll do that and see how we go

**Comment by bentrengrove** [comment]
Looks like it worked! Thank you for doing this!

**Comment by clivejefferies** [comment]
No worries, happy to help.

**Pull request merged**
